src/magnify/apps/core/authentication.py

```diff
@@ -26,9 +26,17 @@ def get_user(self, validated_token):
             for field, oidc_field in settings.JWT_USER_FIELDS_SYNC.items()
             if oidc_field in validated_token
         }
-        # We trust the provider for user authentication
-        defaults.update({"password": "!", "is_active": True})
 
+        # Check audience for known device client
+        audience = validated_token.get("aud")
+        is_device = (
+            audience in getattr(settings, "JWT_USER_DEVICE_AUDIENCES", [])
+            if audience
+            else False
+        )
```

Comment:
I feel that 
```is_device = validated_token.get("aud") in getattr(settings, "JWT_USER_DEVICE_AUDIENCES", [])``` would be more clear and readable.

Reply:
KISS :kissing_heart: